### ISSUE TITLE
Unable to continue without license

### ISSUE THREAD

**Issue by kunaltyagi** [symless/synergy-core#6866]
### Operating Systems ###

Server: Ubuntu 18.04.5

### Synergy Version ###

Synergy 1: `synergys --version`
synergys 1.12.0-snapshot.b1-190b9439

### Steps to Reproduce ###

---BEGIN HELP---

1. Git clone
2. Install dependencies from wiki page (for ubuntu 16.04 and higher)
3. Compile and `sudo make install`
4. Start `synergy` GUI
5. Say no to activation, say no to awesome things
6. Click start
7. Stuck in loop for step 5 through 6

Expected: Server/Client to start
Actual: Stuck in loop

Client from command line works fine

### Log ###

```
QObject::connect: No such slot MainWindow::on_activationDialogFinish()
QObject::connect:   (sender name:   'ActivationDialog')
QObject::connect:   (reciever name: 'MainWindowBase')
```

This is seen on the terminal after step 6

PS: I don't know how to configure from terminal, but I think if the cli is used, the server would also have no issues

**Comment by djjlewis** [comment]
Having the same issue on Arch after synergy was recently update to version 1.13.0-rc2-0badcd0de by my package manager (yay).

**Comment by marlop352** [comment]
also having this issue on a self compiled build (tag v1.13.0-stable) on windows 10 x64

**Comment by marlop352** [comment]
GitHub master has the same problem
tag v1.12.0-stable works fine
tag v1.13.0-rc01 and newer doesn't work properly

last commit confirmed working: 786890dbd2eb057e9cb7c53f52a0d92a272bdc67

I guess something is wrong in the new license code 😅

I can build and run any build to get any logs needed, just point me to the commit and the parameters needed to get the desired logs.

**Comment by kobataiwan** [comment]
> also having this issue on a self compiled build (tag v1.13.0-stable) on windows 10 x64

Enable SYNERGY_ENTERPRISE and the application is working without license

**Comment by marlop352** [comment]
@andreacndn There was no commit to related code, why was it closed?
Is running it in unregistered mode not allowed anymore?
If unregistered mode is not allowed anymore it should be noted on the release notes.

**Comment by andreacndn** [comment]
@marlop352 we restructured the way licenses are managed across a number of commits included in 1.13.0. 
And this behaviour is the result of those changes.
As @kobataiwan as correctly suggested a viable workaround I thought of closing this issue.

Hope this helps,
Andrea

**Comment by marlop352** [comment]
@andreacndn I saw the commits changing how the licenses are managed, that's why I said on my previous message that there were no commits to code related to the issue.

So to clarify, running on unregistered mode is no longer supposed to be allowed by the code?
If it's supposed to be allowed than this is still a bug and the issue should remain open unless the workaround is considered enough (and it seems that this is the case) then please state that the bug won't be fixed or just label it as "won't fix".

On the future when closing a issue with no fix I would suggest stating why it's being closed, whatever the reason is ("workaround considered enough", "too low priority to keep open", "won't be fixed", ...).

Your answer at least points that the project is fine with the proposed workaround.

Thanks for your time,
Marlus

**Comment by kunaltyagi** [comment]
> Enable SYNERGY_ENTERPRISE

@kobataiwan Any links to how? Is it an env variable or a setting or something else?

**Comment by kobataiwan** [comment]
> > Enable SYNERGY_ENTERPRISE
> 
> @kobataiwan Any links to how? Is it an env variable or a setting or something else?

@kunaltyagi  it's a compiler flag. you should re-compile after enable it.

**Comment by Astranox** [comment]
Hi, fedora packager here. I'd like to know if I may use this compile flag for fedora, as currently the GUI tool does not work due to this issue. (Looking at the license I think I may, but I'd like to have a confirmation)

**Comment by andreacndn** [comment]
Hi @Astranox we are working on making available a Community Edition, could we kindly ask you to hold on using this compile flag for now, and watch this space for an update on the matter?

**Comment by Astranox** [comment]
Hi @andreacndn I'd need a solution quite soon (like this week), because as it currently does not work by default I'll have to either downgrade it to Version 12, remove the GUI part from the package or manually remove this specific check or something alike. (Or use this flag anyway, but I don't like doing that without finding a solution together)

**Comment by andreacndn** [comment]
We won't be able to have a solution ready by the end of this week. If it's that urgent I suggest you package 1.12.2, and then we'll updated to the latest at the next opportunity.
Thanks for your open collaboration on this!

**Comment by Astranox** [comment]
For reference, this is the open bug: https://bugzilla.redhat.com/show_bug.cgi?id=1949178

I'll ask if this is a workable solution.

**Comment by kobataiwan** [comment]
> Hi, fedora packager here. I'd like to know if I may use this compile flag for fedora, as currently the GUI tool does not work due to this issue. (Looking at the license I think I may, but I'd like to have a confirmation)

yap, it's compile flag in this project and you could enable it before compile it.

**Comment by paul-learning** [comment]
Hey, sorry I might be too blind - but in the synergy-enterprise.yml I have the following:
` 
 env:
    SYNERGY_ENTERPRISE: "1"
`
is that the right place to set the flag? Or should I do it elsewhere?

**Comment by pol** [comment]
If you are using homebrew, you can download the formula file, update it to add the compile flag, and then install the updated formula file directly with the `-s` flag to avoid using the bottle.

**Comment by dullahan8** [comment]
I just came back to using this after a few years and ran into this while compiling on a Mac following the instructions at https://github.com/symless/synergy-core/wiki/Compiling.

I'm new to brew so can someone help me with the exact command to use so that I can avoid the license prompt?
